docs/running.rst

```diff
@@ -135,6 +135,10 @@ These options may be used with both the GUI and command-line interfaces.
   Enable analyses which increase precision and find more bugs, but which may require more memory and take more time to complete.
   See :doc:`effort`.
 
+-effort[:less|default|more]:
```

Comment:
better to use `effort:[less|default|more]`?

Reply:
Not sure. The less (-precision) and more (+precision) are intermediary levels between min (--precision) and max (++precision). Not sure about default...

Reply:
I just meant that the position of `:` seems strange. 

BTW default value should be `more`, I investigated and noted at http://spotbugs.readthedocs.io/en/latest/effort.html

Reply:
In other words, we have five levels: min, less, default, more and max. And for now default and more has no difference. See `FindBugs` class's `XXX_EFFORT` fields for detail.

Reply:
About the position of the : 
In CLI the context help ouputs:
-effort[:min|less|default|more|max]      set analysis effort level
Will re-write and propose to put together in on entry.